plugin.json

```diff
@@ -5,7 +5,7 @@
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-demo",
     "support_url": "https://github.com/mattermost/mattermost-plugin-demo/issues",
     "icon_path": "assets/icon.svg",
-    "version": "0.10.0",
+    "version": "0.10.1",
```

Comment:
this is currently released version...but displays as v0.10.0
<img width="620" height="168" alt="Screenshot 2025-10-06 at 11 46 36 AM" src="https://github.com/user-attachments/assets/f88c3f89-07b6-45d0-bcac-1880e35daeb1" />

Reply:
I can't reproduce that issue. Did you test with an up-to-date build?
<img width="1311" height="562" alt="Screenshot from 2025-10-07 11-55-15" src="https://github.com/user-attachments/assets/6bec2b6a-21e5-45c9-9c83-bcfab8478f98" />